pgxn/hnsw/hnsw.c

```diff
@@ -132,7 +132,7 @@ hnsw_check_available_memory(Size requested)
 {
 	size_t total;
 	if (sysctlbyname("hw.memsize", NULL, &total, NULL, 0) < 0)
-		elog(ERROR, "Failed to get amount of RAM: %m");
+		elog(ERROR, "Failed to get amount of RAM");
```

Comment:
Why it is necessary to remove"%m" ?
It is valid Postgres blog format character to print system error.
There should been warning here. There is bug in next elog statement where "%n" was wrongly used instead of "%m" (it seems to me that Ihave fixed it long-time ago, but looks like thisPR was not merged)

Reply:
Thanks! Just fixed both places.